### ISSUE TITLE
Main brightness not honored with DDP RGBW

### ISSUE THREAD

**Issue by kfcook** [Aircoookie/WLED#3408]
### What happened?

In version 0.14.0-b6, the main brightness slider (top right of the UI) isn't honored on DDP connected LED strips. It doesn't matter where I set the brightness bar (from 1-255), the brightness stays the same (the brightness will flash a few times, but ultimately stays where it was). When I log into each individual WLED instance, I can control the brightness correctly (i.e. the brightness reacts to the brightness slider), but when trying to control brightness from the DDP controller, it doesn't work.

This worked correctly in previous beta builds.

### To Reproduce Bug

Connect ESP8266 based WLED endpoints to a ESP32 WLED controller using DDP RGBW. Then, try to control the brightness of the virtual strip.

### Expected Behavior

The brightness of the DDP endpoints reacts to the brightness bar on the main controller.

### Install Method

Binary from WLED.me

### What version of WLED?

WLED 0.14.0-b6 (ESP32-D0WDQ6 build 2309241)

### Which microcontroller/board are you seeing the problem on?

ESP8266, ESP32

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by kfcook** [comment]
I just did some experimenting to figure out where this broke. I downloaded the main ESP32 DDP controller to 0.14.0-b3 and everything worked as expected. However, on beta4 and above, the brightness slider is no longer honored. I assume something got broken in the fix for the auto brightness limiter.

**Comment by kfcook** [comment]
I figured out my issue. Since I have brightness limiters set on each individual ESP8266, I had it disabled on the main ESP32 DDP controller. However, when upgrading to b4 or above, it looks like the brightness limiter is automatically re-enabled and set to 850 mA.

I've solved my issue, so I'm going to close this bug out, but it looks like there is a bug in the upgrade process that should be fixed at some point. I don't think there are many of us using DDP, so this likely isn't a priority.
